lib/runner/cli/cli.js

```diff
@@ -454,7 +464,7 @@ class CliRunner {
 
         return promise.then(() => {
           if (this.isConcurrencyEnabled()) {
-            return this.testRunner.runConcurrent(this.testEnvArray, modules)
+            return this.testRunner.runConcurrent(this.testEnvArray, modules, this.isTestWorkersEnabled())
```

Comment:
@gravityvi can't we make `isTestWorkersEnabled` a getter?

Reply:
We can do that. We can do the same for parallel mode as well. But let's change it in another PR

Reply:
sure